cpu/atmega256rfr2/cpu.c

```diff
@@ -96,7 +106,9 @@ ISR(BADISR_vect){
     /* White LED light is used to signal ERROR. */
     LED_PORT |= (BLUE | GREEN | RED);
 
-    while (1) {};
+    while (1) {
```

Comment:
while (1) {} should be OK or better preferred? Just remove semicolon.

see https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions